torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -44,21 +47,43 @@ static IValue Table(
   return Tup(std::move(ivalue_entries));
 }
 
-c10::IValue getFunctionTuple(const Function& func) {
+std::string getModulePath(Node* node) {
+  std::string modulePath = node->scopeName();
+  size_t end = modulePath.size();
+  if (modulePath[end - 1] == '>') {
```

Comment:
I dont quite follow what is happening here. Are you trying to extract just the filename? Also do note that scopeName contains multiple scopes which are all appended together, right? In that case you are touching only the last file name, isnt it? Instead of doing this, why not just save the filename in the first place? Or just keep the whole path.

Reply:
Here, I removed the source range info, which is displayed as `<filename:line:col>`, because we agreed to not display things differently for traced vs. scripted modules. But we can talk more today to figure it out.

Reply:
Add a comment to explain this.